src/Trace/RequestHandler.php

```diff
@@ -289,9 +320,31 @@ private function startTimeFromHeaders(array $headers)
         return null;
     }
 
+    private function nameFromOptions(array $options): string
+    {
+        $rootSpanOptions = array_key_exists('root_span_options', $options)
+                            ? $options['root_span_options']
+                            : array();
+
+        return array_key_exists('name', $rootSpanOptions) ? $rootSpanOptions['name'] : null;
+    }
+
+    private function spanAttrsFromOptions(array $options): array
+    {
+        $rootSpanOptions = array_key_exists('root_span_options', $options)
+                            ? $options['root_span_options']
+                            : array();
+        return array_key_exists('attributes', $rootSpanOptions) ? $rootSpanOptions['attributes'] : array();
+    }
+
     private function nameFromHeaders(array $headers): string
     {
-        return $headers['REQUEST_URI'] ?? self::DEFAULT_ROOT_SPAN_NAME;
+        if (array_key_exists('REQUEST_URI', $headers)) {
+            return strtok($headers['REQUEST_URI'], '?');
```

Comment:
strtok can return bool, resulting in type error

Reply:
`strtok` can return bool only if the first arg is null or empty string. And this [if condition](https://github.com/razorpay/opencensus-php/pull/1/files#diff-215ee834f299135f9e19a5ad11507f15R340) makes sure that doesn't happen.